hypervisor/arch/x86/guest/trusty.c

```diff
@@ -277,7 +277,7 @@ static bool setup_trusty_info(struct acrn_vcpu *vcpu, uint32_t mem_size, uint64_
 	/* Derive dvseed from dseed for Trusty */
 	if (derive_virtual_seed(&key_info.dseed_list[0U], &key_info.num_seeds,
 				  NULL, 0U,
-				  vcpu->vm->uuid, sizeof(vcpu->vm->uuid))) {
+				  (uint8_t *)vcpu->vm->name, strnlen_s(vcpu->vm->name, MAX_VM_NAME_LEN))) {
```

Comment:
char * or uint8* ?

Reply:
we define vm->name to char to avoid type cast for strncmp, and the parameter type is uint8_t* for derive_virtual_seed